tests/test_values.py

```diff
@@ -411,26 +410,23 @@ def test_email_url_value(self):
                 'EMAIL_HOST_PASSWORD': 'password',
                 'EMAIL_HOST_USER': 'user@domain.com',
                 'EMAIL_PORT': 587,
-                'EMAIL_USE_SSL': False,
                 'EMAIL_USE_TLS': True})
-        with env(EMAIL_URL='console://'):
+        with env(EMAIL_URL='consolemail://'):
             self.assertEqual(value.setup('EMAIL_URL'), {
                 'EMAIL_BACKEND': 'django.core.mail.backends.console.EmailBackend',  # noqa: E501
                 'EMAIL_FILE_PATH': '',
                 'EMAIL_HOST': None,
                 'EMAIL_HOST_PASSWORD': None,
                 'EMAIL_HOST_USER': None,
-                'EMAIL_PORT': None,
-                'EMAIL_USE_SSL': False,
-                'EMAIL_USE_TLS': False})
+                'EMAIL_PORT': None})
         with env(EMAIL_URL='smtps://user@domain.com:password@smtp.example.com:wrong'):  # noqa: E501
             self.assertRaises(ValueError, value.setup, 'TEST')
 
     def test_cache_url_value(self):
         cache_setting = {
             'default': {
                 'BACKEND': 'django_redis.cache.RedisCache',
-                'LOCATION': 'redis://host:6379/1',
+                'LOCATION': 'redis://user@host:6379/1',
```

Comment:
Apparently it changes some defaults (more secure by default).

Reply:
I'm not sure if I get your point. Is this change acceptable?

Reply:
It changes defaults apparently, so we should bump the major or at least minor version for it.